WooCommerce/Classes/Blaze/BlazeLocalNotificationScheduler.swift

```diff
@@ -20,6 +20,7 @@ final class DefaultBlazeLocalNotificationScheduler: BlazeLocalNotificationSchedu
     private let pushNotesManager: PushNotesManager
     private var subscriptions: Set<AnyCancellable> = []
     private let blazeEligibilityChecker: BlazeEligibilityCheckerProtocol
+    private var switchStoreUseCase: SwitchStoreUseCaseProtocol?
```

Comment:
Suggestion: It might be worth injecting this in init to unit test that the store switch happens before navigating to campaign creation.

Reply:
Looks great. Thanks, Huong!